frontend/packages/kubevirt-plugin/src/models/templates/vm-template-yaml.ts

```diff
@@ -10,20 +10,21 @@ metadata:
   name: vm-template-example
```

Comment:
shouldn't we rather detect a correct base template and generate it from this? E.g. this is missing cloud init.